src/errors/ValueSetComposeError.ts

```diff
@@ -0,0 +1,8 @@
+import { Annotated } from './Annotated';
+
+export class ValueSetComposeError extends Error implements Annotated {
+  specReferences = ['http://hl7.org/fhir/R4/valueset-definitions.html#ValueSet.compose.include'];
+  constructor(valueSetName: string) {
+    super(`Non-empty ValueSet ${valueSetName} must be composed from at least one inclusion.`);
```

Comment:
Your error message definitely reads a lot better than mine. I would like to still include "non-empty" or something like it in the message, because a ValueSet is allowed to be empty (`compose` has cardinality 0..1). Maybe something like: `ValueSet ${valueSetName} must contain at least one inclusion rule or be empty.`?

Reply:
Oh, right.  I get it now.  So inclusion is only needed if it is not empty... So... what does that really mean?  I'm trying to understand how a FSH user will get in this situation...  Is it basically if they define an exclusion rule but not an inclusion rule?

At any rate, yes, I like your new message.

Reply:
Yes, that's the case that gets the user this error: if there is an exclusion rule, but no inclusion rules. This is the test case that demonstrates this: [log a message when a value set has a logical definition without inclusions](https://github.com/FHIR/sushi/pull/77/files#diff-b8de52dfe4a6614e7f67eca387f5f652R217).